src/ui/app/package.json

```diff
@@ -10,7 +10,7 @@
     "lint:fix": "eslint '*/**/*.{js,ts,tsx}' --format table --fix"
   },
   "dependencies": {
-    "@aqueducthq/common": "0.0.19",
+    "@aqueducthq/common": "0.0.19.alpha",
```

Comment:
before release we are going to want to change this to 0.1.0 to be in sync with the python package. Just want to call that out so we don't forget.

Same thing needs to happen with common of course.